controller/controller_test.go

```diff
@@ -912,7 +912,7 @@ func TestTopologyParams(t *testing.T) {
 				newStorageClass("class-1", "foo.bar/baz"),
 				newClaim("claim-1", "uid-1-1", "class-1", "foo.bar/baz", "", map[string]string{annSelectedNode: "node-1"}),
 			},
-			expectedParams: nil,
+			expectedParams: &provisionParams{},
```

Comment:
Can you add a unit test for this scenario?

Reply:
This case is "node does not exist but there is no other node available", we can add "node does not exist but there is another node available"

Reply:
I think the test case is missing validation that the selected-node annotation is removed.